examples/Torch/NeuralNetsTorchHub/README.md

```diff
@@ -0,0 +1,24 @@
+# Running Inference using a Torch Hub Model
+
+In the following example, you will see how to run inference using a model downloaded directly from [torch hub](https://pytorch.org/hub/). The model we will be using is called the `MiDaS` model, which takes 2D images in RGB space and converts them into 1D images representing depth - more information about this model can be found [here](https://pytorch.org/hub/intelisl_midas_v2/).
+
+### Python Script
+
+This example contains a python script called `torchhub_download.py`. To run this, you need to have a python environment with the pytorch package installed. To install pytorch in python, you can follow the [instructions on the pytorch website](https://pytorch.org/get-started/locally/). Afterwards, you need to change to the example directory and run:
+
+```cmd
+python torchhub_download.py
```

Comment:
```suggestion
uv run torchhub_download.py
```

Consider replacing instructions with a [uv](https://docs.astral.sh/uv/getting-started/installation/)-ready environment so we can make sure all dependencies get installed and the script is run with the above one-liner.